web/src/main/resources/org/openmrs/web/filter/startuperror/header.vm

```diff
@@ -1,6 +1,6 @@
+#parse($includeTemplate)
```

Comment:
Should this parse statement come after the head tag?

Reply:
I have it both ways, so I don't _think_ it matters, however I will change all the headers.vm so that they are consistent.  After I do a quick test, I will commit this change.